config/manifests-multitenant-rhoam/bases/multitenant-managed-api-service.clusterserviceversion.yaml

```diff
@@ -0,0 +1,143 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    operatorframework.io/suggested-namespace: sandbox-rhoam-operator
+    alm-examples: '[]'
+    capabilities: Basic Install
+    categories: "Integration & Delivery"
+    certified: "false"
+    containerImage: quay.io/integreatly/multitenant-managed-api-service:rhoam-v1.1.0
+    operators.operatorframework.io/builder: operator-sdk-v1.2.0
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
+    olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.10"}]'
+    support: RHOAM
+  name: multitenant-managed-api-service.v1.21.0
```

Comment:
Currently installed mt RHOAM is called managed-api-service.v1.20.0. The upgrade to this version will fail because of it